.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -26,6 +26,14 @@ body:
       placeholder: A clear and concise description of the bug.
     validations:
       required: true
+  - type: checkboxes
+    id: regression
+    attributes:
+      label: Regression Issue
+      description: What is a regression? If it worked in a previous version but doesn't in the latest version, it's considered a regression. In this case, please provide specific version number in the report.

```

Comment:
this should probably say that differences from AWS SDK for JavaScript v2 are not regressions

Reply:
> this should probably say that differences from AWS SDK for JavaScript v2 are not regressions

@kuhe Thanks for the feedback. However, the purpose of this checkbox is to identify potential regression in the current major version of SDK, not differences between previous and current major version. The change is implemented across all SDK(s), JS v3 one if them.

**Context:** There was a regression issue in CDK and team implemented this checkbox along with GitHub action that adds `potential-regression` label. It was helpful in triaging such issues quickly and has assisted in identifying some regression issues till date. There is another PR for JS v2. Even though that's getting deprecated, any future security issue fixes or service updates could also cause a regression. Hence I had reopened that PR; @aBurmeseDev will discuss it with JS team.